docs/release-notes.md

```diff
@@ -3,6 +3,32 @@
 - Table of Contents
 {:toc}
 
+## Version 1.9.0
+
+The 1.9.0 release of MCUboot contains various bug fixes, improves
+support on some recent targets, and adds support for devices with a
+write alignment larger than 8.
```

Comment:
But `imgtool` allows only up to 32, not sure larger than this was tested...

Reply:
up to 32 B: https://github.com/mcu-tools/mcuboot/blob/main/boot/bootutil/include/bootutil/bootutil_public.h#L86

Reply:
Thanks, I didn't remember that, we should cite this in the release notes.